### ISSUE TITLE
Rename note with F2 shortcut

### ISSUE THREAD

**Issue by ygcaicn** [vnotex/vnote#1727]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**

Select the note and press the f2 shortcut key to rename it directly (not a pop-up window)

![image](https://user-images.githubusercontent.com/17684070/113088316-efcc9f80-9217-11eb-9da8-e1f7389d4977.png)

**Comment by ennozdd** [comment]
another one is that "Properties" option can be renamed literally to "Rename". It took me a while to figure out this is how you rename a note

**Comment by ygcaicn** [comment]
> another one is that "Properties" option can be renamed literally to "Rename". It took me a while to figure out this is how you rename a note

Yes, but what I want is a more direct approach, similar to how it works in VSC.

**Comment by tamlok** [comment]
Supported
